src/main/java/com/jcabi/maven/plugin/AjcMojo.java

```diff
@@ -198,9 +201,36 @@ public void contextualize(final Context context) throws ContextException {
     @Loggable(value = Loggable.DEBUG, limit = 1, unit = TimeUnit.MINUTES)
     public void execute() throws MojoFailureException {
         StaticLoggerBinder.getSingleton().setMavenLog(this.getLog());
+        final ArtifactHandler artifactHandler = this.project.getArtifact()
+                .getArtifactHandler();
```

Comment:
@mgreau why are you using 8 spaces instead of 4 here and in some lines below? We use 4 everywhere. Please fix.